### PR TITLE
[CMP-1349] Use token instead of user/pass for prometheus connectivity

### PR DESCRIPTION
**Pull request by shantamkumolus**
Developer: @shantamkumolus
Partner: @pgulkari
JIRA - [CMP-1349](https://cloudbolt.atlassian.net/browse/CMP-1349)
Description: Added commands for service account creation in readme file, replaced default service account name with the one provided by user in deployments.yaml

[CMP-1349]: https://cloudbolt.atlassian.net/browse/CMP-1349?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,9 +88,10 @@ export OCP_PORT="<openshift-cluster-port>"
 export OCP_SERVICENAME="<ocp-username>"
 export OCP_ENABLE_SSL_VERIFICATION="<SSL Verification for http and https>"
```
Comment (pgulkari) [inline README.md:89]: Need to remove **OCP_SERVICENAME**

File: README.md [hunk 2]
```diff
@@ -102,7 +115,8 @@ helm install cloudbolt-collector cloudbolt-collector/cloudbolt-collector \
   --set OCP_PORT=$OCP_PORT \
   --set OCP_SERVICENAME=$OCP_SERVICENAME \
```
Comment (pgulkari) [inline README.md:116]: Need to remove **OCP_SERVICENAME**

File: README.md [hunk 3]
```diff
@@ -121,7 +135,8 @@ helm upgrade cloudbolt-collector cloudbolt-collector/cloudbolt-collector \
...
```
Comment (pgulkari) [inline README.md:136]: Need to remove **OCP_SERVICENAME**

File: README.md [hunk 4]
```diff
@@ -134,5 +149,6 @@ helm upgrade cloudbolt-collector cloudbolt-collector/cloudbolt-collector \
...
```
Comment (pgulkari) [inline README.md:150]: Need to remove **OCP_SERVICENAME**

File: values.yaml [hunk 5]
```diff
@@ -9,6 +9,7 @@ OCP_PORT: ""
 OCP_SERVICENAME: ""
```
Comment (pgulkari) [inline values.yaml:9]: Need to remove **OCP_SERVICENAME**

**Review by pgulkari — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shantamkumolus) [reply README.md:88]: OK :+1:

File: README.md (hunk 2 above)
Comment (shantamkumolus) [reply README.md]: OK :+1:

File: README.md (hunk 3 above)
Comment (shantamkumolus) [reply README.md]: OK :+1:

File: README.md (hunk 4 above)
Comment (shantamkumolus) [reply README.md]: OK :+1:

File: values.yaml (hunk 5 above)
Comment (pgulkari) [inline values.yaml:9]: Need to remove this as well @shantamkumolus

**Review by pgulkari — CHANGES_REQUESTED** [review]

File: values.yaml (hunk 5 above)
Comment (shantamkumolus) [reply values.yaml]: OK :+1:

**Review by pgulkari — APPROVED** [review]
Looks good! :+1:

**Pull request merged**